src/sf/grid/social-media/posts.instagram.suma

```diff
@@ -81,13 +81,14 @@ operation MapPost {
 
   return {
     id = post.id
-    url = post.permalink_url
-    createdAt = post.created_time
+    url = post.permalink
+    createdAt = post.timestamp
     text = post.caption
     attachments = [
       {
-        url: post.image_url,
-        title: post.caption
+        url: post.media_url,
+        title: post.video_title || post.caption,
```

Comment:
On second thought I guess it's not a good approach to duplicate the caption, I'd use just the video_title here for now and later used whatever we can get from `children` nodes.

Reply:
I agree I removed it from the map.